configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/OperationReportImpl.java

```diff
@@ -36,10 +37,10 @@ public Result getResult(String key) {
     }
 
     public List<Result> getFailedResults() {
-        return results.values()
+        return Collections.unmodifiableList(results.values()
```

Comment:
No. To do so would require making and filling a new `ImmutableList`; the unmodifiable list that Java natively provides is a decorator that just contains and protects the original collection so it's more efficient when there is already a collection to use.